<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>10</y>
     <width>90</width>
     <height>107</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Dibujar con mouse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonLinea">
      <property name="text">
       <string>Lineas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonCirculo">
      <property name="text">
       <string>Circulos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonEspiral1">
      <property name="text">
       <string>Espiral 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonEspiral2">
      <property name="text">
       <string>Espiral 2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>1261</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Lineas</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>X1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditX1"/>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>X2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditX2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Y1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditY1"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Y2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditY2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Linea DDA</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Linea BRESENHAM</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Circulos</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>196</width>
           <height>111</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>CentroX</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxCentroX">
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>CentroY</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxCentroY">
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Radio</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxRadio">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Circulo Pitagoras</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Circulo Polares</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Espirales</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>196</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>CentroX</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxCentroXEspiral">
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>CentroY</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxCentroYEspiral">
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Radio</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxRadioEspiral">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Factor</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBoxFactorEspiral">
                <property name="maximum">
                 <double>9999.989999999999782</double>
                </property>
                <property name="singleStep">
                 <double>0.500000000000000</double>
                </property>
                <property name="value">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Espiral</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Espiral XC,YC, Radio y Factor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Espirales 2</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>231</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_19">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>CentroX</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxCentroXEspiral2">
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_20">
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>CentroY</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxCentroYEspiral2">
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Th</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBoxThEspiral2">
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                  <property name="singleStep">
                   <double>0.500000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Radio</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBoxRadioEspiral2">
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                  <property name="singleStep">
                   <double>0.500000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Aumento Th</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBoxAumThEspiral2">
                  <property name="minimum">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                  <property name="singleStep">
                   <double>0.500000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_17">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Aumento Radio</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBoxAumRadioEspiral2">
                  <property name="minimum">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>9999.989999999999782</double>
                  </property>
                  <property name="singleStep">
                   <double>0.500000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.900000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Espiral 2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>Espiral 2 Th y Radio</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Fractales</string>
        </property>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>231</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Dibujar</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonDragon1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Dragon </string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonDragon2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>221</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Arabesco</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditX1</tabstop>
  <tabstop>lineEditY1</tabstop>
  <tabstop>lineEditX2</tabstop>
  <tabstop>lineEditY2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
